MC-10462 Add API doc for Reset password in Azure (#83)
cloudops/cloudmc-api-docs: source/includes/azure/_instances.md

------ | -----------
`machineType`<br/>*string* | The new machine type to assign to the instance

<!-------------------- RESET PASSWORD -------------------->

#### Reset password

> The admin password is used to access Linux or Windows [instances](#azure-instances). 

Reviewer: ```suggestion
Reset the administrator password on a Windows or Linux [instances](#azure-instances). 
```

Author: Changed it

> ```shell

Reviewer 2: It should say :
Reset the administrator password on a Windows or Linux [instances] (#azure-instances).

Author: Done!

curl -X POST \
   -H "Content-Type: application/json" \
   -H "MC-Api-Key: your_api_key" \
   -d "request_body" \
   "https://cloudmc_endpoint/v1/services/azure/example/instances/subscriptions/subscriptionId/resourceGroups/cmc-example/providers/Microsoft.Compute/virtualMachines/example-small-server?operation=reset_password"

# Request example:
```

```json
{
  "username": "johndoe",
  "password": "SomePassw0rdVal!d"
}
```

<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances/:id?operation=reset_password</code>

> Update the administrator username for the existing instance, or insert a new user.

Reviewer: ```suggestion
Update the administrator username for the existing instance, or insert a new user if it doesn't exist on the instance.
```

And then maybe we can remove the warning?

Author: Just refactored the entire paragraph.

> <aside class="warning">

Reviewer 2: Not sure if we want to document this

Author: I removed it and added a bigger explanation.

If the username is different than the admin username, it will insert a new user.
</aside>

Required | &nbsp;
------ | -----------

> `username`<br/>*string* | The administrator username of the instance or a new username. It cannot be a reserve user such as admin,root or administrator and must not be more than 20 characters.

Reviewer 3: > It cannot be a **reserved username** admin, root or administrator...

Reviewer 3: missing space before *root*

Author: done in both create instance and reset password.

> `password`<br/>*string* | The password of the account. It must be between between 12 and 72 characters and must be a combination of 3 of the following patterns : Special lettes, Upper Case, Lower Case and Numbers.

Reviewer 3: - What do you mean by `Special lettes`? `Special characters`?
- Upper Case -> Uppercase
- Lower Case -> Lowercase

Author: I copied from create instance and just changed it for both!
